.gitignore

```diff
@@ -12,3 +12,5 @@ target/
 *.class
 *.log
 
+.metals
+.vscode
```

Comment:
(you might like to add those to your [global gitignore](https://stackoverflow.com/questions/7335420/global-git-ignore))